high_card.rb

```diff
@@ -1,5 +1,57 @@
+require "pry"
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+classes = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
+$players = []
+p_1 = $players[0]
```

Comment:
You just assigned an empty array to $players, what are the 1st, 2nd, and 3rd element of that array going to be?